content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

```diff
@@ -164,6 +164,18 @@ Docker's default is the json-file logging driver so the Container Agent tries to
 
 4. Mount the YAML file into your container following the instructions in the [Docker Agent documentation][3]. For more information on setting log drivers for Docker containers, [see this documentation][4].
 
+## Logs are missing but only for some containers
+
+The Docker daemon can have performances issues while it is trying to retrieve logs for containers for which it has already stored large logs files on disk. This could lead to read timeouts when the Datadog Agent is gathering the containers' logs from the Docker daemon. 
+The Datadog Agent outputs a log containing `Restarting reader after a read timeout` when it occurs.
```

Comment:
there are other reasons the Logs agent may log this, generally related to some limitations of the Docker daemon that the Logs Agent tries to work around. Examples I can think of:

* Docker daemon rotating the log file, when the docker log driver's max-file setting is set to 1, leads to a timeout as well. In this example, increasing the timeout on the Logs Agent is actually counter-productive and makes the problem worse.
* container is actually not logging anything for 30+ seconds. I _think_ this leads to a timeout as well (but transparent to the user)

Therefore let's give a more accurate description of the issue: here we could say that the symptoms are that the Logs Agent outputs this message every 30 seconds for a given container and stops sending logs from that container while the container is actually logging messages.